src/components/common/Track/index.tsx

```diff
@@ -10,6 +10,11 @@ type Props = {
   label?: EventLabel
 }
 
+const shouldTrack = (el: HTMLDivElement) => {
+  const disabledChildren = el.querySelectorAll('*[disabled]')
+  return disabledChildren.length === 0
```

Comment:
That's gonna check all nested elements which is not ideal.
Maybe [`closest()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)?

Reply:
`closest` traverses upwards. We want to check the `children`.